### ISSUE TITLE
Import/export does not work

### ISSUE THREAD

**Issue by warpech** [Starcounter/Blending#44]
Import/export fails due to `stardump` not working in Starcounter 2.4.

It fails with the exception:

```
System.EntryPointNotFoundException: Unable to find an entry point named 'star_crud_GetPropertyReadHandle' in DLL 'scdbmetalayer'.
 at Starcounter.Core.Interop.sccrud.star_crud_GetPropertyReadHandle(UInt64 context_handle, String class_name, String property_name, UInt64& property_read_handle)
 at Starcounter.Core.Binding.SubClassBuilder.<>c__DisplayClass38_0.<BuildProperty>b__0()
 at Starcounter.Core.Database.Context.Run(Transaction txn, Action action)
 at Starcounter.Core.Database.Context.Transact(Action action, TransactOptions options)
 at Starcounter.Core.Db.Transact(Action action, TransactOptions options)
 at Starcounter.Core.Binding.SubClassBuilder.BuildProperty(PropertyInfo propInfo)
 at Starcounter.Core.Binding.SubClassBuilder.GenerateImplementation()
 at Starcounter.Core.Binding.ClassBinder.<>c.<BindAllDatabaseClasses>b__0_1(Type t)
 at Starcounter.Core.Database.ProxyMap.RegisterTypesAndLayouts(Dictionary`2 registered, Func`2 factoryMaker)
 at Starcounter.Core.Binding.ClassBinder.BindAllDatabaseClasses()
 at Starcounter.Core.Hosting.Connection.Start()
 at Starcounter.Core.Hosting.DefaultAppHost.Start()
 at StarDump.Core.AppHostHelper.BuildAndStartAppHost(String[] args, Configuration config) in C:\TeamCity\BuildAgent\work\sc-12991\StarDump\src\StarDump.Core\Helpers\AppHostHelper.cs:line 25
 at StarDump.Core.Unload.Run()
 at StarDump.Core.CommandInterface.Unload(Configuration config, Output output) in C:\TeamCity\BuildAgent\work\sc-12991\StarDump\src\StarDump.Core\CommandInterface.cs:line 78
 at StarDump.CommandLineInterface.<>c__DisplayClass9_1.<AddUnloadCommand>b__1() in C:\TeamCity\BuildAgent\work\sc-12991\StarDump\src\StarDump\CommandLineInterface.cs:line 199
 
 C:\Program Files\Starcounter\StarDump>
Settings
```

Depends on https://github.com/Starcounter/StarDump/issues/78

**Comment by PatrykSzwer** [comment]
I've tested our import/export functionality with newest stardump build provided by @miyconst and everything works perfectly! We have to wait until it will be included in SC build, then I'll do final tests and close this issue.

**Comment by warpech** [comment]
@PatrykSzwer, @miyconst what's the status of this?

**Comment by miyconst** [comment]
StarDump is fixed and now works with `2.4.0.xxx`, but the build system is not yet fixed, so the packages is not distributed with the installer. The issue is expected to be fixed during this week.

**Comment by PatrykSzwer** [comment]
I've tested import/export feature with newest SC (2.4.0.5027) and everything worked correctly! I'm closing this issue as solved. @warpech @kegor You can try it by urself, and reopen issue if anything will be wrong for you.
